src/policy/policy.h

```diff
@@ -102,7 +102,9 @@ bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
     /**
      * Check if the transaction is over standard P2WSH resources limit:
      * 3600bytes witnessScript size, 80bytes per witness stack element, 100 witness stack elements
-     * These limits are adequate for multi-signature up to n-of-100 using OP_CHECKSIG, OP_ADD, and OP_EQUAL,
+     * These limits are adequate for multi-signature up to n-of-100 using OP_CHECKSIG, OP_ADD, and OP_EQUAL.
+     *
+     * Also enforce a maximum stack size limit and no annexes for P2TR spends.
```

Comment:
The first half of this comment is confusing. AFAIK, the check enforced L261 is on stack _item_ size not the _stack_ size itself ? 

Few improvements suggestions w.r.t MAX_STANDARD_TAPROOT_STACK_ITEM_SIZE:
* normalize naming compared to MAX_SCRIPT_ELEMENT_SIZE, _item_ or _element_ are designating the same thing ?
* replace TAPSCRIPT with P2TR, akin to usage of of P2WSH/P2WPKH across the codebase
* add the unit (bytes) around variable comment

Reply:
> The first half of this comment is confusing. AFAIK, the check enforced L261 is on a stack item size not the stack size itself ?

Indeed, fixed.

> normalize naming compared to MAX_SCRIPT_ELEMENT_SIZE, item or element are designating the same thing ?

They're not exactly the same; `MAX_SCRIPT_ELEMENT_SIZE` constant is a consensus limit on the size of script pushes, and witness inputs; `MAX_*_STACK_ITEM_SIZE` is a policy rule that only applies to inputs (and isn't relevant during execution). They're indeed confusing, but the issue already exists in the codebase, and I'd rather keep this one analoguous to `MAX_STANDARD_P2WSH_STACK_ITEM_SIZE`.

> replace TAPSCRIPT with P2TR, akin to usage of of P2WSH/P2WPKH across the codebase

They're not the same thing. P2TR is an output type. TAPSCRIPT refers to BIP342 spends of such outputs (meaning P2TR outputs spent using the script path, with a leaf version 0xc0).

> add the unit (bytes) around variable comment

Done.